lib/install/opsuser/opsuser_conf.go

```diff
@@ -39,11 +39,6 @@ var RoleDataCenter = types.AuthorizationRole{
 	Privilege: []string{
 		"Datastore.Config",
 		"Datastore.FileManagement",
-		"VirtualMachine.Config.AddNewDisk",
```

Comment:
Have we tested this with non-robo cases as well? Previously we have needed `VirtualMachine.Config.AddExistingDisk` at the datacenter or cluster level in order to be able to use existing volumes with `run -v existing:/mnt/path`.
Can you confirm we're testing use of existing volumes with restricted ops user in both DRS and non-DRS cases.

Reply:
Looking into this.

Reply:
In the current code, the `VirtualMachine.Config.AddExistingDisk` permission is in the `Endpoint` role, which is assigned to the resource pool in a DRS env and the cluster in a non-DRS env.